openid-4-verifiable-presentations-1_0.md

```diff
@@ -642,40 +642,40 @@ The following is a non-normative example of the payload of a Request Object with
    "response_mode": "direct_post",
    "presentation_definition": {...},
    "nonce": "n-0S6_WzA2Mj",
-   "state" : "eyJhb...6-sVA
+   "state" : "eyJhb...6-sVA"
```

Comment:
```suggestion
   "state": "eyJhb...6-sVA"
```

Looking at this and then at the draft I realized that the JSON examples are also somewhat inconsistent. Some examples seem to be using 3 space indentation (like here), others 2 or 4. Should i create another Issue / PR for this?